pkg/apis/kafka/v1beta1/zz_generated.openapi.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// +build !ignore_autogenerated
```

Comment:
Why this one was removed?

Reply:
That was removed by build, should I put it back?

Reply:
Are you using golang 1.12? If so, keep it this way.

Reply:
go1.12.14